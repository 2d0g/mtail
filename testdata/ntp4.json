[
  {
    "Name": "int_syscalls",
    "Program": "ntpd.mtail",
    "Kind": 0
  },
  {
    "Name": "recvbuf_overflows",
    "Program": "ntpd.mtail",
    "Kind": 0
  },
  {
    "Name": "last_recvbuf",
    "Program": "ntpd.mtail",
    "Kind": 1
  },
  {
    "Name": "exits",
    "Program": "ntpd.mtail",
    "Kind": 0,
    "LabelValues": [
      {
        "Value": {
          "Value": 1,
          "Time": "0000-06-30T12:07:12Z"
        }
      }
    ]
  },
  {
    "Name": "starts",
    "Program": "ntpd.mtail",
    "Kind": 0,
    "LabelValues": [
      {
        "Value": {
          "Value": 1,
          "Time": "0000-06-30T12:04:43Z"
        }
      }
    ]
  },
  {
    "Name": "sync_status",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "LabelValues": [
      {
        "Value": {
          "Value": 1,
          "Time": "0000-12-01T17:42:09Z"
        }
      }
    ]
  },
  {
    "Name": "pll_changes",
    "Program": "ntpd.mtail",
    "Kind": 0
  },
  {
    "Name": "pll_status",
    "Program": "ntpd.mtail",
    "Kind": 1
  },
  {
    "Name": "peer_syncs",
    "Program": "ntpd.mtail",
    "Kind": 0,
    "LabelValues": [
      {
        "Value": {
          "Value": 1,
          "Time": "0000-06-30T12:04:51Z"
        }
      }
    ]
  },
  {
    "Name": "driftfile_errors",
    "Program": "ntpd.mtail",
    "Kind": 0
  },
  {
    "Name": "sync_lost_total",
    "Program": "ntpd.mtail",
    "Kind": 0
  },
  {
    "Name": "peer_status",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_select",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_count",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_code",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_offset",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_delay",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "peer_dispersion",
    "Program": "ntpd.mtail",
    "Kind": 1,
    "Keys": [
      "peer"
    ]
  },
  {
    "Name": "num_peerstats",
    "Program": "ntpd.mtail",
    "Kind": 0,
    "Keys": [
      "peer"
    ]
  }
]
