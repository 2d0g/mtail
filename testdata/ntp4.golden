counter int_syscalls
counter recvbuf_overflows
gauge last_recvbuf
counter exits 1 0000-06-30T12:07:12Z
counter starts 1 0000-06-30T12:04:43Z
gauge sync_status 1 0000-12-01T17:42:09Z
counter pll_changes
gauge pll_status
counter peer_syncs 1 0000-06-30T12:04:51Z
counter driftfile_errors
counter sync_lost_total
gauge peer_status {peer=}
gauge peer_select {peer=}
gauge peer_count {peer=}
gauge peer_code {peer=}
gauge peer_offset {peer=}
gauge peer_delay {peer=}
gauge peer_dispersion {peer=}
counter num_peerstats {peer=}
